### PR TITLE
feat: redirect the user to the prayer camaigns admin page on login

### PR DESCRIPTION
**Pull request by squigglybob**

### CODE REVIEW

**Comment by squigglybob** [comment]
@corsacca what do you think about changing the name of the login button to where it takes you.

So if it takes you to the subscriptions page maybe, "Subscribers" or "Manage subscribers"?

File: porches/generic/site/footer.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
                     </p>
                 </div>
                 <div class="site-info wow fadeInUp" data-wow-duration="1000ms" data-wow-delay="0.3s">
-                    <p><a href="<?php echo esc_html( site_url( '/contacts' ) ); ?>">Login</a> | <a href="<?php echo esc_html( admin_url( 'admin.php?page=dt_prayer_campaigns' ) ); ?>">Page Settings</a></p>
+                    <p><a href="<?php echo esc_html( site_url( '/subscriptions' ) ); ?>">Subscribers</a> | <a href="<?php echo esc_html( admin_url( 'admin.php?page=dt_prayer_campaigns' ) ); ?>">Page Settings</a></p>
```
Comment (corsacca) [inline porches/generic/site/footer.php:34]: Let keep this as "Login"

Comment (squigglybob) [reply porches/generic/site/footer.php:34]: 😄

Comment (squigglybob) [reply porches/generic/site/footer.php]: @corsacca done

**Pull request merged**